cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -272,3 +274,40 @@ def test_provider_refresh_relationship(provider, setup_provider):
     """
     provider.refresh_provider_relationships(method='ui')
     wait_for_relationship_refresh(provider)
+
+
+@pytest.mark.provider([InfraProvider], selector=ONE)
+def test_show_quota_used_on_tenant_screen(request, appliance, provider, setup_provider):
+    """Test show quota used on tenant quota screen even when no quotas are set.
+
+    Steps:
+        1. Navigate to provider's 'Details' page.
+        2. Fetch the information of 'number of VMs'.
+        3. Navigate to 'Details' page of 'My Company' tenant.
+        4. Go to tenant quota table.
+        5. Check whether number of VMs are equal to number of VMs in 'in use' column.
+    """
+    request.addfinalizer(lambda: provider.delete(cancel=False))
+    rhvm_provider = get_crud('rhv41')
+    rhvm_provider.create_rest()
+    request.addfinalizer(lambda: rhvm_provider.delete(cancel=False))
+    view = navigate_to(provider, "Details")
+    provider.refresh_provider_relationships_ui
+    view.browser.refresh
+    view = navigate_to(rhvm_provider, "Details")
+    rhvm_provider.refresh_provider_relationships_ui
+    view.browser.refresh
+    vm_count = rhvm_provider.num_vm()
+    vm_count = vm_count + provider.num_vm()
+    roottenant = appliance.collections.tenants.get_root_tenant()
+    view = navigate_to(roottenant, "Details")
+    row_list = []
+    if appliance.version > "5.9":
+        for row in view.table.rows():
+            row_list.append(row)
```

Comment:
Required:
I know `header index mapping` broken in your example... but
you can go like 
```
In [120]: for row in t:
     ...:     if row[0].text == "Allocated Number of Virtual Machines":
     ...:         vm_count = row[2].text
In [121]: vm_count
Out[121]: u'29 Count'
 ```